debbuild

```diff
@@ -1660,11 +1660,11 @@ sub checkbuildreq {
 	if ($resp !~ /^ok/) {
 	  warn _(' * Required build-dependency ').$pkg.
             _(' is installed, but wrong version (').$reqver.
-            _('):  Need ')."$ver\n"
+            _('):  Need ').$ver."\n";
         }
       } else {
 	# whatever state it's in, it's not completely installed, therefore it's missing.
-	warn _(' * Missing build-dependency ').$pkg."!\n  $pkgdep";
+	warn _(' * Missing build-dependency ')."$pkg!\n  $pkgdep";
```

Comment:
Both 1663 and this line deal with a situation in which a variable is followed by a newline.

1663 changes from interpolation to concatenation. This line chanages from concatenation to interpolation.

What's the reasoning for taking a different approach in each case?

Reply:
Here are two `$variables` placed in a single string. It would be somewhat overkill to split these into three parts.

The single `$variable` in line 1663 is split from `"\n"`, because the two lines above have plain `$variables` as well. Syntax highlighting is more similar.

As I said in the commit comment, this is just slightly more consistent (locally; `debbuild is far too big to polish from start to finish).